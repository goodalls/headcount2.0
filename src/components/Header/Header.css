.header {
  grid-column: 1/7;
  background-color: #273B7A;
  height: 100px;
  display: flex;
  justify-content: center;
  align-items: center; }
  .header h1 {
    font-family: "Slabo 27px", serif;
    color: #E0A449;
    font-size: 2em; }
  .header .icon {
    height: 75px;
    padding-right: 15px; }
