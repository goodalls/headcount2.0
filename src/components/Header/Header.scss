@import '../../variables.scss';

.header {
  grid-column: 1/7;
  background-color: brown;
  height: 100px;
  
  h1 {
    font-family: $header-font;
  }
}

