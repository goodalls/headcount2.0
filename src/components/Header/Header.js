import React from 'react';
import './Header.css';
import image from './school.svg';

const Header = () => {
  return (
    <div className="header">
      <img src={image} alt="Flat icon logo of a school" className="icon" />
      <h1>Headcount 2.0</h1>
    </div>
  );
};

export default Header;
