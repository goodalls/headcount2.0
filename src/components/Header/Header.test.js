import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  let renderedComponent;

  beforeEach(() => {
    renderedComponent = shallow(<Header />);
  });

  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should match the snapshot', () => {
    expect(renderedComponent).toMatchSnapshot();
  });
});
