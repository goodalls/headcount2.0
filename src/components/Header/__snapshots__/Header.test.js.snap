// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="Flat icon logo of a school"
          className="icon"
          src="school.svg"
/>,
        <h1>
          Headcount 2.0
</h1>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Flat icon logo of a school",
          "className": "icon",
          "src": "school.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Headcount 2.0",
        },
        "ref": null,
        "rendered": "Headcount 2.0",
        "type": "h1",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="Flat icon logo of a school"
            className="icon"
            src="school.svg"
/>,
          <h1>
            Headcount 2.0
</h1>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Flat icon logo of a school",
            "className": "icon",
            "src": "school.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Headcount 2.0",
          },
          "ref": null,
          "rendered": "Headcount 2.0",
          "type": "h1",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
